robotbase/src/main/java/frc/robot/commands/pivot/DefaultShooterPivot.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands.pivot;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Robot;
+
+public class DefaultShooterPivot extends Command {
+  public DefaultShooterPivot() {
+    addRequirements(Robot.pivot);
+  }
+
+  @Override
+  public void initialize() {
+    Robot.pivot.setPivotRotations(Constants.SHOOTER_PIVOT.DEFAULT_SHOOTER_PIVOT_SETPOINT);
+  }
+
+  @Override
+  public boolean isFinished() {
+    return false;
+  }
+
+  @Override
+  public void end(boolean interrupted) {
+    Robot.pivot.stop();
+  }
+}
```

Comment:
@NolanCampbell2357 do we want to stop the pivot when this command ends or should we just leave it since any command that interrupts this one will set the setpoint for the shooter pivot?

Reply:
No, for the reason you stated. Additionally, the pivot should function like an arm (It basically is one) so we may need to tune a gravity gain like we did last year for Callaway's arm if the neo cannot hold the setpoint in brake mode. We could stop and start applying output to the motor in the execute method if the neo can hold in brake mode, if we need a gravity gain we will also want to have a setpoint set for the motor.